common/metrics/metric_defs.go

```diff
@@ -1292,6 +1292,8 @@ var (
 	HistoryCount                                  = NewDimensionlessHistogramDef("history_count")
 	SearchAttributesSize                          = NewBytesHistogramDef("search_attributes_size")
 	MemoSize                                      = NewBytesHistogramDef("memo_size")
+	NumPendingChildWorkflowsHigh                  = NewCounterDef("num_pending_child_workflows_high")
+	NumPendingChildWorkflowsTooHigh               = NewCounterDef("num_pending_child_workflows_too_high")
```

Comment:
nit: maybe warn and error? I guess it's very hard to understand the difference without looking at the code that emit those two metrics.